go/v1beta1/storage/storage_test.go

```diff
@@ -24,7 +24,7 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/grafeas/grafeas/go/name"
-	"github.com/grafeas/grafeas/go/v1beta1/api"
```

Comment:
Is this an unused import? If so, remove it.

Reply:
I think this is being used in `func doTestStorage()`